.github/workflows/build_test.yaml

```diff
@@ -51,6 +51,11 @@ jobs:
           cd substrate-node/tests
           robot -d _output_tests/ .
 
+      - name: Go Client tests
+        run: |
+          cd substrate-node/ && ./target/release/tfchain --dev --ws-external
```

Comment:
I'm not sure this will work. Won't it block the CI here?

Reply:
What you can do is test using the robot framework. We could create a robot test that runs the command:
```
go test -v ./...
```
The robot framework already able to start the chain (and wait till it's ready to be used)

Reply:
Well that sounds better!